data/blog/electron-6-0.md

```diff
@@ -91,7 +91,7 @@ In order to package your application correctly with these new helpers you need t
 
  * `shell.showItemInFolder()` now returns void and is an asynchronous call. [#17121](https://github.com/electron/electron/pull/17121)
 
- * Apps must now explicitly set a log path by calling the new function `app.setAppsLogPath()` before using `app.getPath('log')`. [#17841](https://github.com/electron/electron/pull/17841)
+ * Apps must now explicitly set a log path by calling the new function `app.setAppLogPath()` before using `app.getPath('log')`. [#17841](https://github.com/electron/electron/pull/17841)
```

Comment:
too late, but it does should be here `setAppLogsPath`? https://github.com/electron/electron/blob/6-0-x/docs/api/app.md#appsetapplogspathpath